api.swagger.yaml

```diff
@@ -18306,6 +18306,19 @@ components:
             example:
               - 10605
               - 10505
+        confirmed_revision:
```

Comment:
If t's array, maybe better to call it 'confirmed_revisions'?

Reply:
This is normally a single entity will leave it for now as we have already done the react work as well.

But valid point